index.js

```diff
@@ -39,7 +39,7 @@ const assertStore = obj => {
   assert.deepStrictEqual(obj, mapValues(store.getState(), value => value.toJS()))
 }
 
-function addBook (isbn, title) {
+function addBook (isbn = '', title = '') {
```

Comment:
Creative solution but:
- I don't like the default params
- in advanced setups, action creators no longer returns the action directly.